ethereum_history_api/oracles/src/noir/oracles/accountOracle.ts

```diff
@@ -6,9 +6,23 @@ import { decodeAddress, decodeField } from './common/decode.js';
 import { NoirArguments } from './oracles.js';
 import { AlchemyClient } from '../../ethereum/client.js';
 
-const GET_ACCOUNT_ARGS_COUNT = 2;
-const BLOCK_NUM_INDEX = 0;
-const ADDRESS_INDEX = 1;
+export enum ARGS {
+  BLOCK_NUM,
+  ADDRESS,
+  _LENGTH
+}
```

Comment:
I propose to introduce:
```ts
function getEnumSize(arg: object) {
  return object.length /2
}
```

so that later we have:
```ts
export enum ARGS {
  BLOCK_NUM,
  ADDRESS,
  // no _LENGTH
}

  assert(args.length === getEnumSize(ARGS), `get_account requires ${getEnumSize(ARGS)} arguments`);

```

Reply:
Let's just name it `getNumericEnumSize` as this method would be a little different for string enums (no dividing by 2)

Reply:
Found a better one:
```ts
export enum ARGS {
  BLOCK_NUM,
  ADDRESS,
}

class Enum {
  public static size<T extends object>(enumObj: T): number {
      const keys = Object.keys(enumObj).filter(key => isNaN(Number(key)));
      return keys.length;
  }
}



```